config.go

```diff
@@ -366,10 +367,58 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// Exit if you try to use a simulation wallet with a standard
+	// data directory.
+	if cfg.DataDir == defaultDataDir && cfg.CreateTemp {
+		fmt.Fprintln(os.Stderr, "Tried to create a temporary simulation "+
+			"wallet, but failed to specify data directory!")
+		os.Exit(0)
+	}
+
+	// Exit if you try to use a simulation wallet on anything other than
+	// simnet or testnet3.
+	if !cfg.SimNet && cfg.CreateTemp {
+		fmt.Fprintln(os.Stderr, "Tried to create a temporary simulation "+
+			"wallet for network other than simnet!")
+		os.Exit(0)
+	}
+
 	// Ensure the wallet exists or create it when the create flag is set.
 	netDir := networkDir(cfg.DataDir, activeNet.Params)
 	dbPath := filepath.Join(netDir, walletDbName)
-	if cfg.Create {
+
+	if cfg.CreateTemp && cfg.Create {
+		err := fmt.Errorf("The flags --create and --createtemp can not " +
+			"be specified together. Use --help for more information.")
+		fmt.Fprintln(os.Stderr, err)
+		return nil, nil, err
+	}
+
+	if cfg.CreateTemp {
+		tempWalletExists := false
+
+		// Error if the wallet already exists.
```

Comment:
Yeah, I guess it's not really an error (or even needed) as the string below clearly shows it continuing.